croncat/src/config.rs

```diff
@@ -74,7 +71,7 @@ struct RawChainConfigEntry {
 
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct ChainDataSource {
-    pub grpc: String,
+    pub grpc: Option<String>,
```

Comment:
Is this just so things dont error? prolly best to remove

Reply:
:+1: